### ISSUE TITLE
tern exited with code: 1.

### ISSUE THREAD

**Issue by hellboy81** [tststs/atom-ternjs#128]
tern exited with code: 1.
Restart the server via Packages -> Atom Ternjs -> Restart server

```
atom-ternjs
/../.atom/packages/atom-ternjs/node_modules/tern/lib/infer.js:535 var c0 = str.charCodeAt(0) ^ TypeError: Cannot read property 'charCodeAt' of undefined at isInteger (/../.atom/packages/atom-ternjs/node_modules/tern/lib/infer.js:535:17) at extend.hasProp (/../.atom/packages/atom-ternjs/node_modules/tern/lib/infer.js:569:11) at extend.getProp (/../.atom/packages/atom-ternjs/node_modules/tern/lib/infer.js:601:24) at walk.simple.ObjectExpression (/../.atom/packages/atom-ternjs/node_modules/tern/plugin/doc_comment.js:84:44) at c (/../.atom/packages/atom-ternjs/node_modules/tern/node_modules/acorn/dist/walk.js:39:16) at Object.skipThrough (/../.atom/packages/atom-ternjs/node_modules/tern/node_modules/acorn/dist/walk.js:168:3) at c (/../.atom/packages/atom-ternjs/node_modules/tern/node_modules/acorn/dist/walk.js:38:15) at Object.exports.searchVisitor.walk.make.Property (/../.atom/packages/atom-ternjs/node_modules/tern/lib/infer.js:1930:35) at c (/../.atom/packages/atom-ternjs/node_modules/tern/node_modules/acorn/dist/walk.js:38:15) at Object.base.ObjectExpression (/../.atom/packages/atom-ternjs/node_modules/tern/node_modules/acorn/dist/walk.js:302:5)
```

**Comment by agos** [comment]
Same here, restarting does nothing

**Comment by tststs** [comment]
Could you provide a code snippet, so i can reproduce it?

**Comment by mdotasia** [comment]
I'm trying tern for the first time. When I open below Yeoman file I get the same error. Hope you can reproduce the error.

```
'use strict';
var yeoman = require('yeoman-generator');
var chalk = require('chalk');
var yosay = require('yosay');

module.exports = yeoman.generators.Base.extend({

  prompting: {

    askForSiteName: function() {
      var done = this.async();

      this.log(yosay('Welcome to the ' + chalk.red('mdotasia-theme') + ' generator!'));

      var prompts = [{
        type: 'confirm',
        name: 'someOption',
        message: 'Would you like to enable this option?',
        default: true
      }];

      this.prompt(prompts, function (props) {
        this.props = props;
        // To access props later use this.props.someOption;
        done();
      }.bind(this));
    },

    writing: {
      app: function () {
        this.fs.copy(
          this.templatePath('_package.json'),
          this.destinationPath('package.json')
        );
        this.fs.copy(
          this.templatePath('_bower.json'),
          this.destinationPath('bower.json')
        );
      },

      projectfiles: function () {
        this.fs.copy(
          this.templatePath('editorconfig'),
          this.destinationPath('.editorconfig')
        );
        this.fs.copy(
          this.templatePath('jshintrc'),
          this.destinationPath('.jshintrc')
        );
      }
    },

    install: function () {
      this.installDependencies();
    }
  }
});
```

**Comment by tststs** [comment]
Should be fixed with 0.7.4.
